src/pre_defined_classes/mathematical_program/hard/q8/SeriesTest.java

```diff
@@ -0,0 +1,18 @@
+package pre_defined_classes.mathematical_program.hard.q8;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.*;
+
+public class SeriesTest extends BaseTest {
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("The 7th term in the geometric sequence is: 2.44140625E-4")
```

Comment:
I tried to use DoubleLiteral for this but it does not like that there's an E-4 in there. I don't think I can check the math on this but if anyone has any suggestions please give them.

Reply:
You tried using the E-4 in the bounds?

Reply:
That's what the number returned as